queries/javascript/indents.scm

```diff
@@ -0,0 +1,25 @@
+; inherits: (jsx)
+
+[
+  (object)
+  (array)
+  (arguments)
+  (statement_block)
+  (object_pattern)
+  (if_statement)
+  (expression_statement (call_expression))
+  (class_body)
+  (method_definition)
+  (named_imports)
+  (binary_expression)
+  (return_statement)
+] @indent
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
```

Comment:
Indentation issue with else block:
```
if () {
} else {
    }
```
can be fixed by appending @ branch with statement_block

Reply:
that does seem to work, but I found another issue with it. It gives the `else` block a double indentation like this.
```javascript
if (2 > 1) {
  console.log("A")
} else {
    console.log("c")
}
```

Reply:
Really. It seems to work on my end. Let me debug again

Reply:
```
        if (!this.state.toggled) {
            this.props.toggleOn();
        } else {
            this.props.toggleOff();
        }
```
I am getting this kind of indentation. Let me use minimal nvim

Reply:
yep, tried your query and it still doesn't work unfortunately.

Reply:
actually I realized that it works without setting indentxpr and doesnt when indentxpr is explicitly set

Reply:
I don't know why it doesn't set it automatically when `indent.enable` is set to true. I believe it was set automatically in the past but then it stops doing that at some point (or I just misremember it, idk)

Reply:
ahhhh, so that's the issue. Fixed it, thanks!